### DISCUSSION TITLE
162

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#110] [General]
# 潮流周刊第162期 - 晴空塔美

封面图来源于之前在东京玩拍的晴空塔，634m 高的世界第一高塔，远处看着真的很漂亮，刚好那天天气超级好，手机拍摄效果还不错，有一种动漫感。

https://weekly.tw93.fun/posts/162-%E6%99%B4%E7%A9%BA%E5%A1%94%E7%BE%8E

**Comment by LavaCxx** [comment]
咖啡豆可以去淘宝看看辛鹿和柯林，当口粮挺不错的，进阶点可以看看Fisher、Talking、Seesaw这些店。自己做咖啡的好处就是可以品味不同产地、豆种、处理方式的豆的不同风味并找出自己最喜欢的那种。
我两年前还经常做手冲咖啡的时候为此做过[整理](https://glaze-phosphorus-a53.notion.site/45bd5106003c47f9bd24bb9a607e80c9?v=71242ee64f9b470781222c49b7039df0&pvs=4)，可惜从前年买了胶囊机后已经彻底化身懒狗了。

**Reply by tw93** [reply]
哈哈 谢谢推荐

**Comment by coderew** [comment]
这里面的照片都是用什么手机照的呢？

**Reply by tw93** [reply]
15 pro

**Comment by zonemeen** [comment]
大佬，可以尝试尝试蜜处理的咖啡豆🤗

**Reply by tw93** [reply]
可以可以 第一次听这个 「蜜处理的咖啡豆」

**Reply by zonemeen** [reply]
然后推荐来自哥斯达黎加的蜜处理豆

**Comment by JabinHao** [comment]
磨豆机是哪一款呀

**Comment by yslemmo** [comment]
口粮豆直接买illy罐装，便宜，风味稳定
